client/src/pages/Menu.css

```diff
@@ -0,0 +1,59 @@
+.Banner__element {
+  border: 2px none black;
+  padding: 2em 8em;
```

Comment:
here again. if you make the padding smaller, the content will fit the smaller screens too. :)

Reply:
I missed to test it on a smaller screen, but it makes perfect sense. Changing it !